packages/frontend/src/components/Files/CategoryFiles.tsx

```diff
@@ -46,13 +48,14 @@ export const CategoryFiles = ({ childFolderName, nestedFolderId }: TCategoryFile
       const response = await axios.get(
         `/api/alfresco/files?category=${categoryName}${childFolderNameQuery}${nestedFolderIdQuery}&page=${page}`,
       );
-      const { data, hasClassifiedContent } = response.data;
+      const { data, hasClassifiedContent, hasConfidentialContent } = response.data;
```

Comment:
Does this API endpoint in the backend return `hasConfidentialContent` attribute currently?

Reply:
Yep, see commit https://github.com/finnishtransportagency/ratatiedot-extranet/pull/254/commits/2af8b336b57918c2a59475320eda1d666070ff90